doc/source/whatsnew/v1.0.0.rst

```diff
@@ -198,6 +198,7 @@ Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 -
+- ``IndexError`` would not raise if all index values in some index level is missing data (:issue:`20519`)
```

Comment:
Is this MultiIndex-specific? If so, note that. Something like "if all the values in a level of a MultiIndex were missing".